### PR TITLE
added page for SSH keys

### PR DESCRIPTION
**Pull request by jesse-codefresh**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/ssh-config/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/ssh-config/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/ssh-config/docs/

File: _docs/troubleshooting/common-issues/using-ssh-keys.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{% endhighlight %}
+
+
+The pipeline expects a [variable]({{site.baseurl}}/docs/codefresh-yaml/variables/) called “SSH” that you can enter directly or fetch from [shared configuration]({{site.baseurl}}/docs/configure-ci-cd-pipeline/shared-configuration/).
+
+Replace user, host, and command with your own values.
+
+## What to read next
+
+* [How to use SSH keys with GIT](https://codefresh.io/docs/docs/integrations/git-providers/#ssh-keys)
```
Comment (kostis-codefresh) [inline _docs/troubleshooting/common-issues/using-ssh-keys.md:40]: @jesse-codefresh These links are hardcoded. They should be dynamic like those in line 34

File: _docs/troubleshooting/common-issues/using-ssh-keys.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+title: "How to use SSH keys in freestyle steps"
+description: "Running commands remotely from Codefresh Pipeline"
+group: troubleshooting
+sub_group: common-issues
+toc: true
+---
+
+You can easily connect to external servers in Codefresh pipelines and run commands with them via SSH.
+
+First, you need to create or find a Docker image with the SSH client. A good choice is https://hub.docker.com/r/praqma/network-multitool as it has several other networking tools inside.
```
Comment (kostis-codefresh) [inline _docs/troubleshooting/common-issues/using-ssh-keys.md:11]: This should be a URL pointing at https://hub.docker.com/r/praqma/network-multitool

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/ssh-config/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/ssh-config/docs/

**Pull request merged**